sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubMessageSerializerTest.java

```diff
@@ -181,56 +179,60 @@ public void deserializeEventHubProperties() {
         final EventHubProperties properties = serializer.deserialize(message, EventHubProperties.class);
 
         // Assert
-        Assert.assertNotNull(properties);
-        Assert.assertEquals(eventHubName, properties.getName());
-        Assert.assertEquals(createdAt, properties.getCreatedAt());
-        Assert.assertArrayEquals(partitionIds, properties.getPartitionIds());
+        Assertions.assertNotNull(properties);
+        Assertions.assertEquals(eventHubName, properties.getName());
+        Assertions.assertEquals(createdAt, properties.getCreatedAt());
+        Assertions.assertArrayEquals(partitionIds, properties.getPartitionIds());
     }
 
     /**
      * Verify that it throws if the value is not what we expect. In this case, eventHubName is not a string.
      */
-    @Test(expected = AzureException.class)
+    @Test
     public void throwsWhenIncorrectTypeInResponse() {
-        // Arrange
-        final Long eventHubName = 100L;
-        final Date createdAtAsDate  = new Date(1569275540L);
-        final String[] partitionIds = new String[]{ "1", "foo", "bar", "baz" };
+        assertThrows(AzureException.class, () -> {
+            // Arrange
+            final Long eventHubName = 100L;
+            final Date createdAtAsDate  = new Date(1569275540L);
+            final String[] partitionIds = new String[]{ "1", "foo", "bar", "baz" };
 
-        final Map<String, Object> values = new HashMap<>();
-        values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
-        values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, partitionIds);
+            final Map<String, Object> values = new HashMap<>();
+            values.put(ManagementChannel.MANAGEMENT_ENTITY_NAME_KEY, eventHubName);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_CREATED_AT, createdAtAsDate);
+            values.put(ManagementChannel.MANAGEMENT_RESULT_PARTITION_IDS, partitionIds);
 
-        final AmqpValue amqpValue = new AmqpValue(values);
+            final AmqpValue amqpValue = new AmqpValue(values);
 
-        final Message message = Proton.message();
-        message.setBody(amqpValue);
+            final Message message = Proton.message();
+            message.setBody(amqpValue);
 
-        // Act
-        serializer.deserialize(message, EventHubProperties.class);
+            // Act
+            serializer.deserialize(message, EventHubProperties.class);
```

Comment:
Only this line should be in the `assertThrows(() -> {})`, to scope down where we expect the error to be.

Reply:
thanks for the review. changes are incorporated